src/components/Header.tsx

```diff
@@ -27,15 +27,15 @@ export default function Header() {
   }, [location]);
 
   const steps = [
-    'People and date',
-    'Find a place to eat',
-    'Select a timeslot and table',
-    'Reserve your table',
+    'Set the time',
+    'Find the perfect place',
+    'Choose a table',
```

Comment:
Alle anderen Artikel sind definit, nur der hier ist unbestimmt, deswegen würde ich des zu `the` ändern
```suggestion
    'Choose the table',
```

Reply:
Oder wir können alle Artikel zu unbestimmt ändern

Reply:
Ich finde es mit a table irgendwie besser. Hat keinen bestimmten Grund, aber ich finde, es klingt besser

Reply:
äh nein, es mag zwar besser klingen, aber des ist überhaupt nicht consistent.
Entweder wir machen alles zu `a` oder alles zu `the`.
Was haltet ihr denn von
```
Select a time
Find a perfect place
Choose a table
Send your reservation
```

Reply:
Hmm, aber `choose the table` passt einfach nicht. 
Du würdest doch auch nicht sagen `wähle den tisch aus`, sondern `wähle einen tisch aus`

Reply:
Und du würdest auch nich sagen `wähle eine Uhrzeit`, sondern `wähle die Uhrzeit`

Reply:
Ich finde, es ist an der stelle richtig, dass es verschieden ist

Reply:
Aber von mir aus können wir alles zu `a` ändern, das klingt zumindest besser als `the`